internal/worker/worker.go

```diff
@@ -11,13 +11,15 @@ import (
 	"github.com/KyberNetwork/tradelogs/pkg/parser"
 	"github.com/KyberNetwork/tradelogs/pkg/pricefiller"
 	"github.com/KyberNetwork/tradelogs/pkg/storage"
+	"github.com/KyberNetwork/tradinglib/pkg/metrics"
 	ethTypes "github.com/ethereum/go-ethereum/core/types"
 	"go.uber.org/zap"
 )
 
 const (
-	RetryInterval  = 4 * time.Second
-	RemoveInterval = time.Hour
+	RetryInterval          = 4 * time.Second
+	RemoveInterval         = time.Hour
+	ParsingErrorMetricName = "tradelogs_parse_error"
```

Comment:
tradelogs_parse_error
```suggestion
	ParsingErrorMetricName = "trading_tradelogs_parse_error"
```